src/utils/Dashboard.js

```diff
@@ -3,5 +3,5 @@ export function getDashboardHeight() {
 }
 
 export function getDashboardWidth(){
-    return window.innerWidth - 30;
+    return window.innerWidth - (280 + 16 + 72);
```

Comment:
Let us separate these numbers into constants or as part of configuration. This will make it easy to change in the future.